.github/scripts/run-migration.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
To exit immediately with non-zero status when any error occurs:

```suggestion
set -e

```

See also the [docs about GNU Bash's `set` builtin][1].

[1]: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html